metrics/sink_test.go

```diff
@@ -84,60 +70,23 @@ func TestGaugeSink(t *testing.T) {
 }
 
 func TestTrendSink(t *testing.T) {
-	unsortedSamples5 := []float64{0.0, 5.0, 10.0, 3.0, 1.0}
 	unsortedSamples10 := []float64{0.0, 100.0, 30.0, 80.0, 70.0, 60.0, 50.0, 40.0, 90.0, 20.0}
 
 	t.Run("add", func(t *testing.T) {
 		t.Run("one value", func(t *testing.T) {
 			sink := TrendSink{}
 			sink.Add(Sample{TimeSeries: TimeSeries{Metric: &Metric{}}, Value: 7.0})
 			assert.Equal(t, uint64(1), sink.Count)
-			assert.Equal(t, true, sink.jumbled)
 			assert.Equal(t, 7.0, sink.Min)
 			assert.Equal(t, 7.0, sink.Max)
 			assert.Equal(t, 7.0, sink.Avg)
-			assert.Equal(t, 0.0, sink.Med) // calculated in Calc()
 		})
 		t.Run("values", func(t *testing.T) {
 			sink := TrendSink{}
 			for _, s := range unsortedSamples10 {
 				sink.Add(Sample{TimeSeries: TimeSeries{Metric: &Metric{}}, Value: s})
 			}
 			assert.Equal(t, uint64(len(unsortedSamples10)), sink.Count)
-			assert.Equal(t, true, sink.jumbled)
-			assert.Equal(t, 0.0, sink.Min)
-			assert.Equal(t, 100.0, sink.Max)
```

Comment:
The cases `Min`,`Max`, `Avg` on the unsorted array look important. Am I missing a reason why we remove them? :thinking: 

```golang
assert.Equal(t, 0.0, sink.Min)
assert.Equal(t, 100.0, sink.Max)
assert.Equal(t, 54.0, sink.Avg)
```

Reply:
There just have been too many changes overall for me to efficiently keep track of, and I do 3 things at the same time, I lost track or messed it up in a rebase, again 🤦🏻 thanks for pointing it out.

Reply:
Those specific lines have shifted down, I guess the GitHub interface is confusing: https://github.com/grafana/k6/pull/2759/files/179c2dd47ea83b8f6563e83019108f1b898035c8#diff-f37293b2fc466b351eccf12f450b485548f0ca27048f28327602c752f8b2db24R90

But I guess it's also true that with removing `jumbled` I lost a bit track along the way, I'll add a test back to make sure  `sorted` and not sorted are tested 👍🏻

Reply:
I've brought back the initial test on the `sorted` state, thanks for bringing that up 👍🏻 

Plus, I've added a couple of others to assert that the `Add` and `P` operations do modify it 🚀

Reply:
Sorry, my bad :cry: Thanks! :bow: